tests/jballerina-unit-test/src/test/resources/test-src/annotations/deprecation_annotation.bal

```diff
@@ -297,3 +297,26 @@ function myFunction(int i, string s) {
 function testUsingDepricatedFunction() {
     myFunction(1, "hello");
 }
+
+type Employee record {|
+    @deprecated
+    string name;
+    int id;
+    @deprecated
+    Job job;
+|};
+
+type Job record {|
+    string title;
+    @deprecated
```

Comment:
Do we have tests for the valid case with deprecation documentation?

Reply:
Are referring to this?  https://github.com/ballerina-platform/ballerina-lang/pull/37178/files/c2b9f6389b852f52cff6ce1e378a635b8c574bdb#diff-7c9e0755b817109ef54edf03b366fb416bccb11775ef563c4fe45be37413f2caR101

Reply:
Please see https://github.com/ballerina-platform/ballerina-lang/pull/37178#discussion_r954774117

Reply:
I will add tests with valid cases

Reply:
Updated https://github.com/ballerina-platform/ballerina-lang/commit/3fc30db5fbc84e389a635fa4c84d2b913b7b6035